typing_extensions is useless when we have all the types in the latest python version already
--
diff --git a/zulipterminal/api_types.py b/zulipterminal/api_types.py
index 17dc1a1..bdf73d6 100644
--- a/zulipterminal/api_types.py
+++ b/zulipterminal/api_types.py
@@ -1,6 +1,6 @@
 from typing import Any, Dict, List, Optional, Union
 
-from typing_extensions import Literal, TypedDict
+from typing import Literal, TypedDict
 
 # These are documented in the zulip package (python-zulip-api repo)
 from zulip import EditPropagateMode  # one/all/later
diff --git a/zulipterminal/config/keys.py b/zulipterminal/config/keys.py
index 079fae5..452d806 100644
--- a/zulipterminal/config/keys.py
+++ b/zulipterminal/config/keys.py
@@ -1,7 +1,7 @@
 from collections import OrderedDict
 from typing import List
 
-from typing_extensions import TypedDict
+from typing import TypedDict
 from urwid.command_map import (
     CURSOR_DOWN,
     CURSOR_LEFT,
diff --git a/zulipterminal/config/markdown_examples.py b/zulipterminal/config/markdown_examples.py
index b52cc74..f46842a 100644
--- a/zulipterminal/config/markdown_examples.py
+++ b/zulipterminal/config/markdown_examples.py
@@ -1,6 +1,6 @@
 from typing import List
 
-from typing_extensions import TypedDict
+from typing import TypedDict
 
 
 class MarkdownElements(TypedDict):
diff --git a/zulipterminal/config/ui_mappings.py b/zulipterminal/config/ui_mappings.py
index 781597a..9fbc748 100644
--- a/zulipterminal/config/ui_mappings.py
+++ b/zulipterminal/config/ui_mappings.py
@@ -1,6 +1,6 @@
 from typing import Dict, Optional
 
-from typing_extensions import Literal
+from typing import Literal
 
 from zulipterminal.api_types import EditPropagateMode
 from zulipterminal.config.symbols import (
diff --git a/zulipterminal/core.py b/zulipterminal/core.py
index 3ea88dc..6e8168d 100644
--- a/zulipterminal/core.py
+++ b/zulipterminal/core.py
@@ -13,7 +13,7 @@ from typing import Any, Dict, List, Optional, Tuple, Type, Union
 import pyperclip
 import urwid
 import zulip
-from typing_extensions import Literal
+from typing import Literal
 
 from zulipterminal.api_types import Composition, Message
 from zulipterminal.config.themes import ThemeSpec
diff --git a/zulipterminal/helper.py b/zulipterminal/helper.py
index 69021ab..be4edab 100644
--- a/zulipterminal/helper.py
+++ b/zulipterminal/helper.py
@@ -26,7 +26,7 @@ from typing import (
 from urllib.parse import unquote
 
 import requests
-from typing_extensions import TypedDict
+from typing import TypedDict
 
 from zulipterminal.api_types import Composition, EmojiType, Message
 from zulipterminal.config.keys import primary_key_for_command
diff --git a/zulipterminal/model.py b/zulipterminal/model.py
index d9452ac..95355ff 100644
--- a/zulipterminal/model.py
+++ b/zulipterminal/model.py
@@ -22,7 +22,7 @@ from urllib.parse import urlparse
 
 import zulip
 from bs4 import BeautifulSoup
-from typing_extensions import Literal, TypedDict
+from typing import Literal, TypedDict
 
 from zulipterminal import unicode_emojis
 from zulipterminal.api_types import (
diff --git a/zulipterminal/platform_code.py b/zulipterminal/platform_code.py
index f633605..5e4d7e3 100644
--- a/zulipterminal/platform_code.py
+++ b/zulipterminal/platform_code.py
@@ -1,7 +1,7 @@
 import platform
 import subprocess
 
-from typing_extensions import Literal
+from typing import Literal
 
 
 # PLATFORM DETECTION
diff --git a/zulipterminal/ui_tools/boxes.py b/zulipterminal/ui_tools/boxes.py
index 12f8a53..4a4dd58 100644
--- a/zulipterminal/ui_tools/boxes.py
+++ b/zulipterminal/ui_tools/boxes.py
@@ -11,7 +11,7 @@ import dateutil.parser
 import urwid
 from bs4 import BeautifulSoup
 from bs4.element import NavigableString, Tag
-from typing_extensions import Literal
+from typing import Literal
 from tzlocal import get_localzone
 from urwid_readline import ReadlineEdit
 
diff --git a/zulipterminal/ui_tools/buttons.py b/zulipterminal/ui_tools/buttons.py
index 6813f1d..d53678b 100644
--- a/zulipterminal/ui_tools/buttons.py
+++ b/zulipterminal/ui_tools/buttons.py
@@ -4,7 +4,7 @@ from typing import Any, Callable, Dict, List, Optional, Tuple, Union, cast
 from urllib.parse import urljoin, urlparse
 
 import urwid
-from typing_extensions import TypedDict
+from typing import TypedDict
 
 from zulipterminal.api_types import RESOLVED_TOPIC_PREFIX, EditPropagateMode
 from zulipterminal.config.keys import is_command_key, primary_key_for_command
diff --git a/zulipterminal/ui_tools/views.py b/zulipterminal/ui_tools/views.py
index e8aec55..b49118e 100644
--- a/zulipterminal/ui_tools/views.py
+++ b/zulipterminal/ui_tools/views.py
@@ -5,7 +5,7 @@ from typing import Any, Callable, Dict, List, Optional, Sequence, Tuple, Union
 
 import pytz
 import urwid
-from typing_extensions import Literal
+from typing import Literal
 
 from zulipterminal.api_types import EditPropagateMode
 from zulipterminal.config.keys import (
